src/util/log.rs

```diff
@@ -222,3 +305,31 @@ macro_rules! fatal {
         }
     })
 }
+
+fn color_if_atty(color: &str, isatty: bool) -> &str {
```

Comment:
Minor nit -- `isatty(2)` is short for `is a tty` (a `tty` being the abbreviation for "teletype" device).  Might want to rename this to `color_if_tty`.

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2234/commits/7219536b3a6293a74ae0439c681ea4446bebd64c